content/concepts/tokens/non-fungible-token-transfers.md

```diff
@@ -0,0 +1,82 @@
+
+
+## Buy Offers
+
+
+### Create a Buy Offer
+
+Any account can offer to buy a `NFToken` on the XRP Ledger. You can create a buy offer using `NFTokenCreateOffer` with a _Flags_ setting of _null_. You provide the _Owner_ account, _NFTokenID_, and the _Amount_ of your offer.
+
+
+### Accept a Buy Offer
+
+Use the `NFTokenAcceptOffer` transaction to transfer a `NFToken`. Provide the `nft_offer_index` and the owner account to complete the transaction.
+
+
+## Trading Modes
+
+When trading `NFTokens`, you can choose between a _direct_ transaction between a buyer and seller or a _brokered_ transaction, where a third party account matches sell and buy offers to arrange the trade.
+
+Trading in direct mode gives the seller control over the transfer. The seller can either post `NFTokens` for purchase by anyone, or sell NFTokens to a specific destination account. The seller receives the entire purchase price for the `NFToken`.
+
+In brokered mode, the seller allows a third party account to broker the sale of the `NFToken`. The broker account collects a broker fee for the transfer at an agreed upon rate. The purchase is completed in real time, paying the broker and seller from the buyer’s funds without requiring an up front investment by the broker.
+
+
+### When to Use Brokered Mode
+
+If a `NFToken` creator has the time and patience to seek out the right buyers, the creator keeps all proceeds from the sale. This works fine for a creator who sells few `NFTokens` at variable prices.
+
+On the other hand, creators might not want to spend their time selling their creations when they could spend the time creating. Instead of handling each individual sale, the sales work can be turned over to a third-party broker account.
+
+Using a broker offers several advantages. For example:
+
+
+
+* The broker can act as an agent, working to maximize the selling price of the `NFToken`. IF the broker is paid a percentage of the sale price, the higher the price, the more the broker can earn.
+* The broker can act as a curator, organizing `NFTokens` based on a niche market, price point, or other criteria. This can attract groups of buyers who might not otherwise discover a creator’s work.
+* The broker can act as a marketplace, such as Opensea.io, to handle the auction process at the application layer.
+
+
+### Brokered Sale Workflows
+
+In the most straightforward workflow, a creator mints a new `NFToken`. The creator initiates a sell offer, entering the minimum acceptable sale price and setting the broker as the destination. Potential buyers make bids for the `NFToken`, setting the broker as the destination for the bid. The broker selects a winning bid and completes the transaction, taking a broker’s fee. The broker then cancels any remaining buy offers for the `NFToken`.
```

Comment:
I don't completely understand the chain of custody from this explanation.

When the broker "completes sale using NFTokenAcceptOffer" does the broker actually become the owner of the NFToken at any point? Or is setting the broker as the destination just a technical detail that allows them to use the NFTokenAcceptOffer to direct the ownership of the token to the right bidder?

Reply:
Also, can the bidders cancel their own offers if the broker doesn't do so? (I would imagine that they have to be able to—otherwise lazy/malicious brokers could leave people with dangling objects consuming their reserve, yeah?)

Reply:
The broker never takes possession of the NFToken. They find a buyer and complete the transaction, transferring the NFToken from the seller to the buyer. The funds go directly from the buyer to the seller, less the broker fee, which goes directly from the buyer to the broker at the time of the sale. 

Bidders can cancel their own buy offers (and should), but best practices are for the brokers to cancel all of the losing offers for the reasons you mention.